ci/build_wheel.sh

```diff
@@ -56,7 +56,19 @@ fi
 
 cd "${package_dir}"
 
-python -m pip wheel . -w dist -vvv --no-deps --disable-pip-version-check
+
+PARALLEL_LEVEL=$(python -c \
+  "from math import ceil; from multiprocessing import cpu_count; print(ceil(cpu_count()/2))")
```

Comment:
we recently added a `PARALLEL_LEVEL` environment variable to the `rapids-configure-sccache` script below:

- https://github.com/rapidsai/gha-tools/blob/dcd134953586fbcbe9924131e07fe443088da0a5/tools/rapids-configure-sccache#L7

That script is `source`d earlier in this file.

Therefore you could simply use the value defined from that script instead of redefining it here.

Reply:
Great. Just wanted to be aggressive here to rule out the impact of hyperthreading. Using all 64 logical cores to compile cugraph did fail at times on my workstation (w/ Threadripper 3975WX).

Reply:
I don't think hyperthreading is a factor on these CI machines -- we get the "real" number of cores, afaik. We should be safe to remove this. Let's wait to push until after CI runs on the current commit (I'm testing [something else](https://github.com/rapidsai/cugraph/pull/4489#discussion_r1643469633) at the moment).